packages/dapp/src/components/Dashboard/ActivityList.tsx

```diff
@@ -37,12 +42,38 @@ class ActivityList extends React.Component<ActivityListOwnProps> {
               />
             );
           })}
+        {!!this.props.resolvedChallenges &&
+          this.props.appealChallenges &&
+          this.props.appealChallenges.map(c => {
+            index++;
+            return (
+              <ResolvedAppealChallengeActivityListItem
+                toggleSelect={this.props.toggleChallengeSelect!}
+                key={c}
+                challengeID={c!}
+                even={index % 2 === 0}
+              />
+            );
+          })}
         {!this.props.resolvedChallenges &&
           this.props.challenges &&
           this.props.challenges.map(c => {
             index++;
             return <ChallengeActivityListItem key={c} challengeID={c!} even={index % 2 === 0} />;
           })}
+        {!this.props.resolvedChallenges &&
+          this.props.appealChallenges &&
+          this.props.appealChallenges.map(c => {
+            index++;
+            return (
+              <ResolvedAppealChallengeActivityListItem
```

Comment:
Should this be `AppealChallengeActivityListItem`?

Reply:
as far as I can tell the only difference between the 2 components is that one supports the check-box. I think that we will end up removing the "Resolved" version of both components during this sprint's refactor but wanted to leave it in for now to sorta line up with the non-appeal-challenge versions